src/wp-includes/blocks/footnotes.php

```diff
@@ -211,4 +211,77 @@ function wp_add_footnotes_to_revision( $fields ) {
 function wp_get_footnotes_from_revision( $revision_field, $field, $revision ) {
 	return get_metadata( 'post', $revision->ID, $field, true );
 }
-add_filter( 'wp_post_revision_field_footnotes', 'wp_get_footnotes_from_revision', 10, 3 );
+add_filter( '_wp_post_revision_field_footnotes', 'wp_get_footnotes_from_revision', 10, 3 );
+
+/**
+ * The REST API autosave endpoint doesn't save meta, so we can use the
+ * `wp_creating_autosave` when it updates an exiting autosave, and
+ * `_wp_put_post_revision` when it creates a new autosave.
+ *
+ * @since 6.3.0
+ *
+ * @param int|array $autosave The autosave ID or array.
+ */
+function _wp_rest_api_autosave_meta( $autosave ) {
```

Comment:
Instead of doing this, can we not just change the autosave endpoint?

Reply:
Using hooks felt to me like a more isolated temporary solution while we wait for general meta support. I'm not sure how I feel about hardcoding updating footnotes meta into the endpoint. Or do you mean filtering/overriding the autosave controller from this file? Might also be more prone to breakage if someone changes the controller?